src/server_manager/electron_app/build.action.sh

```diff
@@ -40,8 +60,13 @@ cp -r "${BUILD_DIR}/server_manager/web_app/static" "${STATIC_DIR}/server_manager
 # is loaded via a custom protocol.
 cp src/server_manager/package.json package-lock.json "${STATIC_DIR}"
 cd "${STATIC_DIR}"
-npm install --prod --ignore-scripts
+npm ci --prod --ignore-scripts
 
 # Icons.
 cd "${ROOT_DIR}"
 electron-icon-maker --input=src/server_manager/images/launcher-icon.png --output=build/server_manager/electron_app/static
+
+# shellcheck disable=SC2046
+electron-builder $(node src/server_manager/scripts/get_electron_build_flags.mjs "${PLATFORM}" --buildMode "${BUILD_MODE}")
```

Comment:
We call Node to get the parameters for electron-builder. We may consider calling electron-builder from Node as well. There's an API: https://www.electron.build/api/electron-builder
It doesn't have to be in this PR.

Reply:
This seems like a great thing to do as we move all these scripts to node